figures/scripts/figS2-snv-callers-panels.R

```diff
@@ -135,30 +131,27 @@ for (dataset in c("tcga", "pbta")) {
 
   ## Upset plots -------------------------------------------------------
   print("Preparing data for upset plot")
-  detect_mat <- all_caller_df %>%
-    # Bring over VAF columns and the index
+  vaf_mat <- all_caller_df %>%
+    # Bring over VAF columns
     select(starts_with("VAF_")) %>%
     as.matrix()
 
-  # Store the indices as dimnames
-  dimnames(detect_mat)[[1]] <- all_caller_df$index
```

Comment:
I think I might have left this, though I can't quite remember why we wanted it.

Reply:
Well, the `index` column is gone now, so this line is a bug unless I go back and revert removing the `index` column..

Reply:
Right, but I figured you would catch that!